cloud-nio/cloud-nio-impl-drs/src/main/scala/cloud/nio/impl/drs/DrsConfig.scala

```diff
@@ -16,9 +17,9 @@ final case class DrsConfig(marthaUrl: String,
 object DrsConfig {
   // If you update these values also update Filesystems.md!
   private val DefaultNumRetries = 3
-  private val DefaultWaitInitial = 30.seconds
-  private val DefaultWaitMaximum = 5.minutes
-  private val DefaultWaitMultiplier = 2.0d
+  private val DefaultWaitInitial = 10 seconds
+  private val DefaultWaitMaximum = 30 seconds
+  private val DefaultWaitMultiplier = 1.5d
   private val DefaultWaitRandomizationFactor = 0.1
```

Comment:
These values are actually not relevant in this PR since the inner retry is not invoked through the engine code; I dialed them back anyway b/c they seemed quite long.

Reply:
They were long because due to WA-90 Martha v2 & v3 already re-hammers servers when they report that they are overloaded. Shortening the time here shrinks the amount of time given to the servers to recover, and possibly finishing a restart if they've crashed. For example, I think when we knocked over JDR in the past it took around 2 minutes for the application to restart.

That said, I don't have any current empirical evidence to support NOT changing these numbers, so LGTM

Reply:
I'm sure there was good reason for these values in the past, and that reason may not have changed. The thing that tips me over to agreeing to this change is that _not_ making this change could put Cromwell (or at least other workflows) in jeopardy, which feels like the wrong trade-off for being _very_ patient with DRS providers falling over and having to restart. Shortening this helps protect Cromwell while putting the pressure on other services to behave more reasonably.

Ideally, we could be patient _and_ not jeopardize overall Cromwell health, but that's an effort for another day.